gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
Hmm. I changed this because it gives IntelliJ a lot of help when working on the build scripts (it'll nag you to switch by default). There'll be a small download time penalty in CI, yes. Probably only a few seconds... worth it?